src/components/Navigation/Navigation.Footer.tsx

```diff
@@ -23,8 +23,7 @@ const siteQuery = graphql`
       }
     }
     allMdx(
-      sort: { fields: frontmatter___date, order: ASC }
-      filter: { frontmatter: { date: { ne: null } } }
```

Comment:
Somehow this causes gatsby to return nothing. as a quick fix, added a hack to filter articles with null date,
https://github.com/sudaraka94/gatsby-theme-novela/pull/1/files#diff-dcf38e325404620bd39513b370c500bc178c777ce607fdb837d82ab499c3f03fR44